cli/man/splinter-authid.1.md

```diff
@@ -48,11 +48,14 @@ SUBCOMMANDS
 : Shows a specific authorized identity on a Splinter node
 `create`
```

Comment:
These should be alphabetical

Reply:
Fixed - moved to first commit with this file